modal_proto/api.proto

```diff
@@ -1856,6 +1856,15 @@ message Proxy {
   string proxy_id = 5;
   repeated ProxyIp proxy_ips = 4;
 }
+message ProxyCreateRequest {
+  string deployment_name = 1 [ (modal.options.audit_target_attr) = true ];
+  DeploymentNamespace namespace = 2;
```

Comment:
Can we get rid of DeploymentNamespace? We're using it for global objects, but I think we should perhaps deprecate that unless we really need it

Reply:
I think we should. It's not useful for proxies. Dicts, Mounts, Queues, and Proxies take the `namespace` parameter in `from_name()`. Do you think I should remove it now?

Reply:
Removed it from both the proto and `Proxy.from_name()`. :)